examples/tutorials/analysis-time/light_curve.py

```diff
@@ -201,13 +201,18 @@
 lc_maker_3d = LightCurveEstimator(
     energy_edges=[1, 10] * u.TeV, source="crab", reoptimize=False
 )
+# Example showing how to change some parameters from the object itself
+lc_maker_3d.n_sigma_ul = 3              # Number of sigma to use for upper limit computation
+lc_maker_3d.selection_optional = "all"  # Add the computation of upper limits and likelihood profile
 lc_3d = lc_maker_3d.run(analysis_3d.datasets)
 
 
 ######################################################################
 # The LightCurve object contains a table which we can explore.
 # 
 
+# Example showing how to change just before plotting the threshold on the signal significance (points vs upper limits)
+lc_3d.sqrt_ts_threshold_ul = 16
```

Comment:
Note that this is sqrt(TS), so you effectively apply a threshold of 16 sigma (!) here. I guess you rather mean 4?
```suggestion
lc_3d.sqrt_ts_threshold_ul = 4
```